scripts/search-job-messages.py

```diff
@@ -20,28 +20,31 @@
 
 LIMIT = 42
 
+
+
+
 args = sys.argv
-sumo = SumoLogic(args[1], args[2])
```

Comment:
This is a breaking change, let's not do it for this PR. You can file a new issue and we'll have a different PR for it.

Reply:
I reverted the breaking change.